web_src/css/base.css

```diff
@@ -374,6 +374,13 @@ a.label,
 
 .ui.selection.dropdown .menu > .item {
   border-color: var(--color-secondary);
+  white-space: nowrap;
+}
+
+.ui.search.dropdown > .text {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
Why only "search dropdown" has these styles, but all ".ui.dropdown .menu > .item" also have "text-overflow: ellipsis" ?

Reply:
Not sure if that's the best selector but it was one of the matching ones in this case. The classname on it is certainly some kind of abuse `ui search normal selection dropdown`.

Reply:
I removed the `search` class from the selector, but would not do any more change to this currently. Actually I'm not sure I understand the question.

Reply:
OK, I think the problem is clear now. We can't "overflow: hidden" to any general "dropdown" selector.

Because even `.ui.selection` or `.ui.search` it might contain images like avatars.


Ref:  Broken avatar css on main page #30624